[Running] python -u "/home/chad/Downloads/gradiento_paaiskinimas(1).py"
Traceback (most recent call last):
  File "/home/chad/Downloads/gradiento_paaiskinimas(1).py", line 23, in <module>
    pavirsius(xx,yy,zz)
  File "/home/chad/Downloads/gradiento_paaiskinimas(1).py", line 10, in pavirsius
    ax = fig.gca(projection='3d')
TypeError: FigureBase.gca() got an unexpected keyword argument 'projection'

[Done] exited with code=1 in 0.994 seconds

[Running] python -u "/home/chad/Downloads/gradiento_paaiskinimas(1).py"
Traceback (most recent call last):
  File "/home/chad/Downloads/gradiento_paaiskinimas(1).py", line 23, in <module>
    pavirsius(xx,yy,zz)
  File "/home/chad/Downloads/gradiento_paaiskinimas(1).py", line 10, in pavirsius
    ax = fig.gca(projection='3d')
TypeError: FigureBase.gca() got an unexpected keyword argument 'projection'

[Done] exited with code=1 in 0.793 seconds

[Running] python -u "/home/chad/Downloads/uni.py"
[1, 2, 3]

[Done] exited with code=0 in 0.021 seconds

[Running] python -u "/home/chad/Downloads/uni.py"
[1, 2, 3]

[Done] exited with code=0 in 0.021 seconds

[Running] python -u "/home/chad/Downloads/uni.py"
[1, 2, 3]

[Done] exited with code=0 in 0.022 seconds

[Running] python -u "/home/chad/Downloads/uni.py"
[1, 2, 3]

[Done] exited with code=0 in 0.022 seconds

[Running] python -u "/home/chad/Downloads/uni.py"
[1, 2, 3]

[Done] exited with code=0 in 0.033 seconds

[Running] python -u "/home/chad/Downloads/uni.py"
[1, 2]
[1, 2, 3]
[1, 2, 3]

[Done] exited with code=0 in 0.022 seconds

[Running] python -u "/home/chad/Downloads/uni.py"
[1, 2]
[1, 2, 3]
[1, 2, 3]

[Done] exited with code=0 in 0.021 seconds

[Running] python -u "/home/chad/Downloads/uni.py"
[1]
[1, 2]
[1, 2, 3]
[1, 2, 3]

[Done] exited with code=0 in 0.023 seconds

